README.md

```diff
@@ -4,9 +4,15 @@ Studion Platform common infra components.
 
 ## Table of Contents
 
-1. [Installation](#installation)
-2. [Usage](#usage)
-3. [API](#api)
+1. [Prerequisites](#prerequisites)
+2. [Installation](#installation)
+3. [Usage](#usage)
+4. [API](#api)
+
+## Prerequisites
+
+- Working [Pulumi](https://www.pulumi.com/docs/get-started/) project
```

Comment:
Maybe we should link this page https://www.pulumi.com/docs/clouds/aws/get-started/begin/#pulumi-aws-before-you-begin, what do you think? It describes how to install pulumi and set aws access key and secret.

Reply:
I agree